content/blog/2021/2021-01-29-how-to-use-the-aws-cli-to-find-untagged-instances/index.md

```diff
@@ -0,0 +1,180 @@
+---
+layout: post
+title: "How to use the AWS CLI to find untagged instances"
+date: 2021-01-29
+comments: true
+author: Onica team
```

Comment:
Other blogs like this have used "Rackspace Onica Team" for the author, so might want to change this to be consistent

Reply:
updated!!